### PR TITLE
Clarify exercises for stream setup; closes #209

### LINKED ISSUE #209: Exercises need clarification for the stream-first setup
Also the instructor guide requires instructions on how to prepare and coordinate the exercises.

### PR DESCRIPTION
**Pull request by bast**
Working on the second exercise ...

### CODE REVIEW

**Comment by bast** [comment]
IMO can be merged but just leaving it out here for suggestions/comments.

File: content/centralized.md [hunk 1]
```diff
@@ -49,47 +49,63 @@ Real life examples:
...
+  - Then **everyone in your group** needs their GitHub account to be added as collaborator to the exercise repository:
+    - Participants give their GitHub usernames to their chosen administrator (in their respective group, in online workshops you can use the Zoom chat for private communication within the breakout room).
+    - Administrator gives the other group members the newly created GitHub repository URL.
+    - Administrator adds participants as collaborators to their project (Settings → Manage Access → Invite a collaborator).
+  ````
+
+  ````{tab} Participating via stream
+  We create(d) these the day before hopefully:
+  - <https://github.com/cr-workshop-exercises/centralized-workflow-exercise> (this will not be shown on stream)
+  - <https://github.com/cr-workshop-exercises/centralized-workflow-exercise-recorded> (this will be shown on stream and recorded)
```
Comment (rkdarst) [inline content/centralized.md:78]:
```suggestion
  - <https://github.com/cr-workshop-exercises/centralized-workflow-exercise-recorded> (this will be shown on stream and recorded, **your username and comments may appear in the recorded video on YouTube**)
```

File: content/centralized.md [hunk 2]
```diff
@@ -49,47 +49,63 @@ Real life examples:
...
+``````{prereq} Exercise preparation
+`````{tabs}
+  ````{tab} Part of team/ exercise room
+  - In-person: We form small groups (4-5 persons). Video: use breakout rooms.
+  - Each group needs to appoint someone who will host the shared GitHub repository: *an administrator*.
+    This is typically the exercise lead (if available).
+  - **One person per group (administrator)** generates a new repository
+    from the template <https://github.com/coderefinery/template-centralized-workflow-exercise>
+    called `centralized-workflow-exercise` (There is no need to tick *"Include all branches"* for this exercise):
+    ```{figure} img/centralized/generate_repo.png
```
Comment (rkdarst) [inline content/centralized.md:65]: red arrow pointing to "generate this template" ?  To be added in separate PR.

File: content/centralized.md [hunk 3]
```diff
@@ -49,47 +49,63 @@ Real life examples:
...
+    ```
+  - Then **everyone in your group** needs their GitHub account to be added as collaborator to the exercise repository:
+    - Participants give their GitHub usernames to their chosen administrator (in their respective group, in online workshops you can use the Zoom chat for private communication within the breakout room).
+    - Administrator gives the other group members the newly created GitHub repository URL.
+    - Administrator adds participants as collaborators to their project (Settings → Manage Access → Invite a collaborator).
+  ````
+
+  ````{tab} Participating via stream
+  We create(d) these the day before hopefully:
+  - <https://github.com/cr-workshop-exercises/centralized-workflow-exercise> (this will not be shown on stream)
```
Comment (rkdarst) [inline content/centralized.md:77]:
```suggestion
  - <https://github.com/cr-workshop-exercises/centralized-workflow-exercise> (this will not be shown on stream or recorded in our videos, but is be public on the Internet until it is deleted)
```

File: content/centralized.md [hunk 4]
```diff
@@ -263,13 +287,13 @@ At the same time, helpers can review open pull requests from their exercises gro
 - We recommend that pull requests are reviewed by someone else in your group.
 - In our example everyone has write access to the "central" repository.
 
-**Code review and protected branches**
+**Protected branches**
 
 - A good setting is to make the `master` or `main` branch **protected** and all changes to it have to go
```
Comment (rkdarst) [inline content/centralized.md:292]:
```suggestion
- A good setting for large projects is to make the `master` or `main` branch **protected** and all changes to it have to go
```
"large" isn't quite the right word here but maybe is the most useful to say?

File: content/distributed.md [hunk 5]
```diff
@@ -73,22 +73,33 @@ Please step out of the repository and check that you fork the **forking**-workfl
...
+    called `forking-workflow-exercise`
+  - In this case we **do not add collaborators** to the repository (this is the point of this example).
+  - Share the link to the newly created repository in the shared document with your group.
+  
+  **Learners in team/breakout-room**: Fork the helper's newly created repository and clone the fork.
+  ````
+
+  ````{tab} Participating via stream
+  We create(d) these the day before hopefully:
+  - <https://github.com/cr-workshop-exercises/forking-workflow-exercise> (this will not be shown on stream)
```
Comment (rkdarst) [inline content/distributed.md:92]:
```suggestion
  - <https://github.com/cr-workshop-exercises/forking-workflow-exercise> (this will not be shown on stream or recorded in our videos, but is be public on the Internet until it is deleted)
```

File: content/distributed.md [hunk 6]
```diff
@@ -73,22 +73,33 @@ Please step out of the repository and check that you fork the **forking**-workfl
...
+  - <https://github.com/cr-workshop-exercises/forking-workflow-exercise-recorded> (this will be shown on stream and recorded)
```
Comment (rkdarst) [inline content/distributed.md:93]:
```suggestion
  - <https://github.com/cr-workshop-exercises/forking-workflow-exercise-recorded> (this will be shown on stream and recorded, **your username and comments may appear in the recorded video on YouTube**)
```

**Review by rkdarst — APPROVED** [review]

File: content/centralized.md (hunk 2 above)
Comment (bast) [reply content/centralized.md:65]: Good idea

**Comment by bast** [comment]
Thanks for the very good clarifications!
